app/models/instrument_for_cart.rb

```diff
@@ -0,0 +1,18 @@
+class InstrumentForCart < ProductForCart
+
+  def purchasable_by?(acting_user, session_user)
+    super.tap do |success|
+      if acting_user.blank?
+        @error_path = url_helpers.new_user_session_path
+      elsif product.schedule_rules.none?
+        @error_message = text(".schedule_not_available", i18n_params)
+      end
+
+      # If an instrument is not purchasable, we always redirect. If ProductForCart
+      # didn’t set an error path due to some specific error, we should redirect
```

Comment:
I have the codeclimate browser plugin and it's complaining about using only ascii in comments. I usually wouldn't care if it were a specific reason (an accented character or whatnot), but this looks like it's just a curly apostrophe.

![screen shot 2018-05-21 at 1 28 43 pm](https://user-images.githubusercontent.com/1099111/40323465-1052c194-5cfb-11e8-9d01-5436054fe484.png)

Reply:
I think that’s a terrible rubocop check, so I always mark it as invalid. Would you like me to change the apostrophe here?